### DISCUSSION TITLE
Please help setup Https

### DISCUSSION THREAD

**Discussion by btcuserone** [btcpayserver/btcpayserver#3228] [Technical Support]
I setup btcpay server on lunanode with my own domain but the Https does not work 

please help me solve this, i bought ssl cert i uploaded it to the server and successfully updated the ca-certificate  but https still does not work

its setup on ubutnu 20.04 + ngix

thank you for any support

**Comment by pavlenex** [comment]
Please open a support ticket with lunannode, it's beyond our control. You do not need to buy an SSL, it's issues for free via let's encrypt when you deploy.

**Comment by Spixmaster** [comment]
I probably know what your issue is if your server has the certificate installed properly. The solution is described here: https://github.com/btcpayserver/btcpayserver/issues/2322.
It has nothing to do with a broken certificate but rather with a missing HTTP header.

**Comment by btcuserone** [comment]
thank you for the responses

i solved this rather quickly via lunanode support, the issue was on the DNS side, had to configure an additional DNS for the same url but without the www.
